feature/settings/src/main/res/values/strings.xml

```diff
@@ -101,6 +101,11 @@
     <string name="fps_stabilization_disclaimer">*Available stabilization modes may change due to selected frame rate.</string>
     <string name="lens_stabilization_disclaimer">*Some devices may not support stabilization on both lens.</string>
 
+    <!-- disabled reason strings-->
+    <string name="device_unsupported">%1$s is unsupported by the device</string>
+    <string name="fps_unsupported"> %1$s is unsupported at %1$d fps</string>
```

Comment:
I think this should be `%1$d` -> `%2$d`. I get a format conversion exception otherwise.